log/log.go

```diff
@@ -71,6 +71,34 @@ func InitLogger(logLevel Level) {
 	}
 }
 
+// InitLoggerJSON initializes global logger with full json format.
+//
+// The JSON format is also compatible with logdna's ingestion format:
+// https://docs.logdna.com/docs/ingestion
+//
+// Pass in debug as log level enables development mode,
+// which makes DPanicLevel logs panic.
+func InitLoggerJSON(logLevel Level) {
```

Comment:
🔕 Should we use this as the default `Init` rather than `InitLogger`?

Reply:
The default one will be controlled by `InitFromConfig` from https://github.com/reddit/baseplate.go/pull/168, eventually :)